src/plugins/discover/public/application/components/sidebar/discover_field.scss

```diff
@@ -2,3 +2,7 @@
   min-width: 260px;
```

Comment:
I'm not sure why we need to do this...but if we're keeping it let's use Eui variables:

```
.dscSidebarItem__fieldPopoverPanel {
  min-width: $euiSizeXXL * 6.5;
  max-width: $euiSizeXXL * 7.5;
}
```